CHANGELOG.md

```diff
@@ -1,115 +1,94 @@
-## Main
-* [6189](https://github.com/grafana/loki/pull/6189) **irizzant**: Add creation of a `ServiceMonitor` object for Prometheus scraping through configuration parameter `create_service_monitor`. Simplify mixin usage by adding (https://github.com/prometheus-operator/kube-prometheus) library.
+## Main/Unreleased
+
+### All Changes
+
+#### Loki
+
+##### Enhancements
+* [6361](https://github.com/grafana/loki/pull/6361) **chaudum**: Add new unwrapped range aggregation `rate_counter()` to LogQL
```

Comment:
This PR is already in 2.6: https://github.com/grafana/loki/pull/6615/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR75

```suggestion
```

The PR number was incorrect here - [slack thread](https://raintank-corp.slack.com/archives/C029V4SSS9L/p1656606128601149?thread_ts=1656600038.825129&cid=C029V4SSS9L)

Reply:
done